oas

```diff
@@ -343,164 +60,94 @@
+      "TrackDataModel": {
+        "title": "TrackDataModel",
         "type": "object",
-        "required": [
-          "type",
-          "coordinates"
-        ],
         "properties": {
-          "type": {
-            "pattern": "^Polygon$",
-            "type": "string"
+          "identifier": {
+            "title": "Identifier",
+            "type": "string",
+            "default": "12:34:56:78:90:AB"
           },
-          "coordinates": {
-            "description": "See https://json.schemastore.org/geojson.json"
+          "SSCC": {
+            "title": "Sscc",
+            "type": "string",
+            "default": "00512345678916851849"
+          },
+          "GSIN": {
+            "title": "Gsin",
+            "type": "string",
+            "default": "40212345678945787992"
+          },
+          "data": {
+            "title": "Data",
+            "type": "array",
+            "items": {
+              "type": "object"
+            },
+            "default": [
+              {
+                "timestamp": "2022-07-13T13:30:30.778575",
+                "temperature": 29.5,
+                "humidity": 50.1,
+                "pressure": 1020,
+                "gps": {
+                  "lat": 10.23,
+                  "long": 43.54
+                }
+              }
+            ]
           }
         },
-        "example": {
-          "type": "Polygon",
-          "coordinates": [
-            [
-              [
-                11.029761135578156,
-                49.57058013297548
-              ],
-              [
-                11.030300259590149,
-                49.57058013297548
-              ],
-              [
-                11.030300259590149,
-                49.57087931601995
-              ],
-              [
-                11.029761135578156,
-                49.57087931601995
-              ],
-              [
-                11.029761135578156,
-                49.57058013297548
-              ]
-            ]
-          ]
-        }
+        "description": "Data model for the data response"
       },
-      "Error": {
-        "description": "This defines the structure of an error payload when one is present. Actual messages and additional error information content may change from one implementation to another",
-        "type": "object",
+      "ValidationError": {
+        "title": "ValidationError",
         "required": [
-          "message"
+          "loc",
+          "msg",
+          "type"
         ],
+        "type": "object",
         "properties": {
-          "message": {
-            "type": "string",
-            "description": "Error description.",
-            "example": "General description"
-          },
-          "errors": {
-            "description": "Additional detailed information",
+          "loc": {
```

Comment:
Why is there a location in a ValidationError?

As a general rule, please use the same type of format for errors as other service templates (see field_data for example)